### ISSUE TITLE
"unable to complete your request"

### ISSUE THREAD

**Issue by IliaStu** [razvanmarinescu/brain-coloring#27]
Dear Mr Razvan Marinescu:
I am writing to ask whether you could help me. When loading .csv into BrainPainter, an error occurs. "The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application."

**Comment by razvanmarinescu** [comment]
Yes. The server is down as we're migrating it to Google Colab. Use the source code in the meantime.

**Comment by IliaStu** [comment]
What does source code mean? I'm a doctor so I don't know much about programming.

**Comment by razvanmarinescu** [comment]
Aah. Sorry, it means to install the software locally via the command line (terminal). But don't worry, in that case you'll need to wait ~2weeks until we finalize the new notebook in Google Colab. Feel free to send me a reminder.
